tests/foreman/api/test_contentmanagement.py

```diff
@@ -114,13 +114,14 @@ def test_positive_sync_kickstart_repo(self, module_manifest_org, default_sat):
 
         :BZ: 1687801
         """
+        distro = 'rhel8'
```

Comment:
Yes, it is supposed to be there. It was not my intention to parametrize this test (though it would be possible), I just wanted to keep the original functionality.